src/applications/simple-forms/form-upload/containers/ConfirmationPage.new.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ConfirmationPageView } from '../../shared/components/ConfirmationPageView.v2';
+
+const content = {
+  headlineText: 'Your form has been submitted and is pending processing',
+  nextStepsText: (
+    <p>
+      Right now your form is pending processing. We’ll review your submission
+      and contact you if we have any questions.
+    </p>
+  ),
+};
+
+const ConfirmationPage = () => {
+  const form = {
+    data: { veteran: { fullName: { first: 'John', last: 'Veteran' } } },
+    submission: {
+      timestamp: 1721999975,
+      response: { confirmationNumber: '2106a7e8-27dd-46b4-886a-29b780723f4c' },
```

Comment:
Is this `form` supposed to have mock data in our `form-upload` application?

Reply:
Oops, this was accidentally added to the wrong directory.

Reply:
Fixed!